adaptors/adaptors.pro

```diff
@@ -39,6 +39,19 @@ SUBDIRS += humidityadaptor
 SUBDIRS += pressureadaptor
 SUBDIRS += temperatureadaptor
 
+config_hybris {
```

Comment:
Would it make sense to create a local variable (for example HYBRIS_SUBDIRS) at the top of the file with the content below and then assign/append that to SUBDIRS here and above in the contains(CONFIG,hybris) condition? This would cause less duplication in the file.

Reply:
Done.